client/flutter/lib/components/page_scaffold.dart

```diff
@@ -7,34 +7,37 @@ class PageScaffold extends StatelessWidget {
 
   final List<Widget> body;
 
-  final BuildContext context;
```

Comment:
This context was unused and also not a good practice to pass it to Widgets in this way, so removed.